.github/workflows/integration-tests.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
+      - run: git config --global core.longpaths true
```

Comment:
to fix Filename too long error on Windows
https://github.com/autifyhq/autify-cli/actions/runs/4592136184/jobs/8108956931?pr=389